lib/shared/types/src/types/apis/sdk/serverSDKTypes.ts

```diff
@@ -0,0 +1,86 @@
+import { DVCLogger, DVCDefaultLogLevel } from '../../../logger'
+import { DVCReporter } from '../../../reporter'
+
+/**
+ * Options to control the setup of the DevCycle NodeJS Server SDK.
+ */
+export interface DevCycleServerSDKOptions {
+    /**
+     * Logger override to replace default logger
+     */
+    logger?: DVCLogger
+
+    /**
+     * Metrics reporter to capture data about event processing
+     */
+    reporter?: DVCReporter
+
+    /**
+     * Set log level of the default logger
+     */
+    logLevel?: DVCDefaultLogLevel
+
+    /**
```

Comment:
does it really make sense to share this type around to all the different server packages. It feels like many of these options are irrelevant to the regular js server SDK, and shouldn't show up in the autocomplete of someone using it

Reply:
what about things like enableCloudBucketing and all the stuff related to event flushing and config refetching?